src/com/facebook/buck/intellij/ideabuck/src/com/facebook/buck/intellij/ideabuck/build/BuckBuildManager.java

```diff
@@ -235,4 +235,15 @@ public void runInCurrentThread(
       final BuckCommandHandler handler, @Nullable final Runnable postStartAction) {
     handler.runInCurrentThread(postStartAction);
   }
+
+  /**
+   * Run handler in the current thread with a post end action.
+   *
+   * @param handler a handler to run
+   * @param postEndAction an action that is executed
+   */
+  public void runInCurrentThreadPostEnd(
```

Comment:
This is needed since I can't write settings on the thread buck query returns to.